notebooks/automatic_embedding_tei_inference_endpoints.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<p>Is the user you in this case? Maybe, clarify a little. E.g. "I'm uploading the dataset to the account of the user I logged in with, but....."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>